platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/PreconsensusEventReplayWorkflow.java

```diff
@@ -72,8 +72,8 @@ public static void replayPreconsensusEvents(
             @NonNull final QueueThread<GossipEvent> intakeQueue,
             @NonNull final ConsensusRoundHandler consensusRoundHandler,
             @NonNull final QueueThread<ReservedSignedState> stateHashSignQueue,
-            @NonNull final StateManagementComponent stateManagementComponent,
-            final long initialMinimumGenerationNonAncient) {
+            final long initialMinimumGenerationNonAncient,
+            @NonNull final Supplier<ReservedSignedState> latestImmutableState) {
```

Comment:
missing param doc for `latestImmutableState`

Reply:
added